docs/topics/mta-rn-known-issues-7-0-3.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-3_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.3 has the following known issues.
+
+.IntelliJ IDE plugin Keymap actions do not work
+// Scheduled to be fixed in 7.0.3. Move this to resolved issues once fixed?
+
+In order to automate {ProductShortName}, the IntelliJ IDE plugin Keymap actions must work. They currently do not work.  link:https://issues.redhat.com/browse/MTA-2460[(MTA-2460)]
+
+.Eclipse IDE cannot run {ProductShortName} analysis
+
+// Get more information
+Eclipse IDE cannot run {ProductShortName} plugin and perform analysis. link:https://issues.redhat.com/browse/MTA-2563[(MTA-2563)]
```

Comment:
Are there more details that would be appreciated by the user? Now this description just sort of copies what is already said in the RN title. 

```suggestion
Eclipse IDE cannot run the {ProductShortName} plugin and perform application analysis. link:https://issues.redhat.com/browse/MTA-2563[(MTA-2563)]
```